tests/chart/legendSpec.js

```diff
@@ -0,0 +1,116 @@
+(function(d3, fc) {
+    'use strict';
+
+    describe('legend', function() {
+
+        var element;
+
+        beforeEach(function() {
+            element = document.createElement('div');
+        });
+
+        function verifyLegend(tableData) {
+            expect(element.children[0].nodeName).toEqual('TABLE');
+
+            var table = element.children[0];
+            expect(table.children.length).toEqual(tableData.length);
+
+            for (var i = 0; i < tableData.length; i++) {
+                var row = table.children[i];
+                expect(row.children[0].innerHTML).toEqual(tableData[i][0]);
+                expect(row.children[1].innerHTML).toEqual(tableData[i][1]);
+            }
+        }
+
+        it('should render the given items', function() {
+            var legend = fc.chart.legend()
+                .items([
+                    ['open', function(d) { return d.open; }],
+                    ['close', function(d) { return d.close; }]
+                ]);
+
+            var datum = {
+                open: 45.67,
+                close: 56.78
+            };
+
+            d3.select(element).data([datum])
```

Comment:
Minor point: missing newline and I've been preferring datum for cases like this (single node selection).